polybar-scripts/info-healthchecks/README.md

```diff
@@ -0,0 +1,29 @@
+# Script: info-healthcheck
```

Comment:
Maybe 'info-healtcheck.io' is a better script name.

Reply:
You mean 'info-healthchecks.io'? Other info scripts for online services do not do that either. I can see that this make sense since "healthcheck" is more general, but this would introduce "." into the directory name which may cause confusion.

Reply:
We have  [network-ipinfo.io](https://github.com/polybar/polybar-scripts/tree/master/polybar-scripts/network-ipinfo.io).
But just `info-healthcheck` would assume that this is something independent which is not the case.

Reply:
Fair point. I've changed the name.